src/index.ts

```diff
@@ -88,7 +88,15 @@ export class Transport {
 	/**
 	 * Closes the connection.
 	 */
-	public async close() {
+	public async close(options: { flush?: boolean } = { flush: false }) {
```

Comment:
Discussed further with @mwoodffdc and @sfryffdc . We came to the conclusion that Orizuru should **always flush the queues before closing the connection**, and that the previous behaviour should be considered a defect.
Can you please remove this options parameter and make the function always close the channels?
You'll need to rework the unit tests to ensure you get full coverage of the positive and negative `if` cases.

Reply:
Removed.